tests/test_image_conversion.py

```diff
@@ -6,115 +6,108 @@
 import xarray as xr
 
 import imagej.dims as dims
+import imagej.images as images
 
-# -- Fixtures --
+# -- Image helpers --
 
 
-@pytest.fixture(scope="module")
 def get_img(ij_fixture):
-    def _get_img():
-        # Create img
-        CreateNamespace = sj.jimport("net.imagej.ops.create.CreateNamespace")
-        dims = sj.jarray("j", [5])
-        for i in range(len(dims)):
-            dims[i] = i + 1
-        ns = ij_fixture.op().namespace(CreateNamespace)
-        img = ns.img(dims)
-
-        # Populate img with random data
-        cursor = img.cursor()
-        while cursor.hasNext():
-            val = random.random()
-            cursor.next().set(val)
-
-        return img
-
-    return _get_img
-
-
-@pytest.fixture(scope="module")
-def get_imgplus():
-    def _get_imgplus(ij_fixture):
-        """Get a 7D ImgPlus."""
-        # get java resources
-        Random = sj.jimport("java.util.Random")
-        Axes = sj.jimport("net.imagej.axis.Axes")
-        UnsignedByteType = sj.jimport(
-            "net.imglib2.type.numeric.integer.UnsignedByteType"
-        )
-        DatasetService = ij_fixture.get("net.imagej.DatasetService")
+    # Create img
+    CreateNamespace = sj.jimport("net.imagej.ops.create.CreateNamespace")
+    dims = sj.jarray("j", [5])
+    for i in range(len(dims)):
+        dims[i] = i + 1
+    ns = ij_fixture.op().namespace(CreateNamespace)
+    img = ns.img(dims)
```

Comment:
Yeah it looks like this works! The reason its currently done with `CreateNamespace`  is because of the `ij.op().create()` ambiguity issue (https://github.com/imagej/pyimagej/issues/106).

I like your suggestion more. It is easier to read/understand. https://github.com/imagej/pyimagej/commit/ea9fc13c4c5b83149ad5715eee366a055120b69c

Reply:
Gotcha, good to know that it's completed and that we can write something simpler!